### ISSUE TITLE
Comparing Different models

### ISSUE THREAD

**Issue by mohammadshahabuddin** [Onr/Council-GAN#13]
Hi,
Could you please tell me how you compared different models? Did you use the same learning rate, number of epochs, Number of decay epochs, image size, optimizer among all models? Also, did you collect test results using the final saved generator or did you use the best results testing all saved generators at different epochs?

**Comment by Onr** [comment]
Hi Mohammad ,
if the model have an available checkpoint for that task we used that, starGAN if i remember doesn't, but it have an instruction on how to make one. For the ones that don't we try using what the authors did and trained with the same image size and number of epochs. for MUNIT which our code relies on we just changed the config file losses to match that of MUNIT we also trained with there original parameters which got lower scores,
For our methode we just used the last  epoch

**Comment by mohammadshahabuddin** [comment]
Hi,

Thank you for the information! It will be really helpful.

Thanks
Shahab
